content/en/dashboards/querying/_index.md

```diff
@@ -33,17 +33,18 @@ When you first open the graphing editor, you are on the **Edit** tab. Here, you
 To configure your graph on dashboards, follow this process:
 
 1. [Select the visualization](#select-your-visualization)
-2. [Choose the metric to graph](#choose-the-metric-to-graph)
-3. [Filter](#filter)
-4. [Aggregate and rollup](#aggregate-and-rollup)
-5. [Apply additional functions](#advanced-graphing)
-6. [Title the graph](#create-a-title)
+2. [Define the metric](#Define-the-metric)
```

Comment:
```suggestion
2. [Define the metric](#define-the-metric)
```
Since the `#choose-the-metric-to-graph` heading is changed, the reference links throughout the docs also have to change, for example: https://github.com/DataDog/documentation/blob/aea233bf21d7a2e1391e0b48245588a6761e7d06/content/en/dashboards/guide/quick-graphs.md?plain=1#L69

Let me know if you have questions on this or if you'd like me to commit the reference changes.